oeps/oep-0027-course-metadata-authoring.rst

```diff
@@ -0,0 +1,151 @@
+  .. _edX subdomains: https://openedx.atlassian.net/wiki/spaces/AC/pages/663224968/edX+DDD+Bounded+Contexts#edXDDDBoundedContexts-edXSubdomainMap
+
+
+Decisions
+---------
+
+* We will no longer embed workflow into the core system. Rather, we will develop or enhance existing APIs to offload workflow and business process to systems and services that allow for more agile changes.
+
+* Conceptually, we will separate the notion of Course within the edX system in the following ways:
+
+  * **Course Learning Content**, a course-team created package of learning material, with its weekly and sub-weekly structure
+    (i.e., Sections, Subsections, Units, etc.) and learning blocks (i.e., leaf xBlocks), tied to specific dates of offering.
+     
+    * This is what we call **(Studio) Course Run**, for the time being.
+    * It is bound to the organization that authors the content.
+    * Note: In the future, this concept may be further separated when we support content reuse across "Course Runs".
+      At that time, we may distinguish between:
+
+      * **Reusable Learning Content** (not tied down to specific dates nor structure) and
+      * **Packaged Learning Content** (an "instantiation" of a "Reusable Learning Content" with structure, dates, and 
+        configuration).
+
+  * **Discoverable Course**, a learner-facing representation of a course that entices enrollment with marketing information
+    such as the course's price and credit/degree outcomes.
+
+    * This is what we call **Catalog Course**, for the time being.
+    * It is bound to a specific marketing site.
+
+  * **Learner Consumable Course**, a learner-engaging course, with read-optimized representation of the learning content. 
+
+    * This is what we call **(LMS) Course Run**.
+    * It is bound to a specific learner.
+
+* Given the above distinctions, each `edX subdomain`_ will manage its own concept. Hence, the authoring logic
+  implementation for each concept will also remain separate. This prevents us from needing to maintain these concepts across
+  multiple domains: keep the authoring of Discoverable-Course (**Catalog Course**) within Discovery; keep the authoring of 
+  Learning-Content-Course (**Course Run**) within Content Authoring. For example, concepts of price, marketing-publish-date,
+  marketing sales, discounts, etc., do not need to bleed into the Content Authoring service.
+
+* The following is Course Run metadata and its editing should be owned by the `Content Authoring subdomain`_:
+
+  * course run dates
+
+    * start
+    * end
+    * enrollment start
+    * enrollment end
+    * certificate availability
+
+  * pacing type
+
+* The following is Catalog Course metadata and its editing should be owned by the `Discovery subdomain`_:
+
+  * course about image
+  * instructors
+  * seat price
+  * seat type
+  * other catalog course marketing data
+
```

Comment:
Since neither of these lists are exhaustive, I'm left wondering where other metadata ends up, such as description, and subject. If that's meant to be documented elsewhere, can you include a link?